src/core_plugins/kibana/public/discover/components/field_chooser/lib/detail_views/string.html

```diff
@@ -44,7 +43,7 @@ <h5 ng-show="!field.details.error">Quick Count <kbn-info info="Top 5 values base
   <a
     ng-href="{{field.details.visualizeUrl}}"
     ng-show="field.visualizable"
-    class="kuiButton kuiButton--primary kuiButton--small kuiButton--fullWidth kuiVerticalRhythmSmall"
+    class="kuiButton kuiButton--secondary kuiButton--small kuiButton--fullWidth kuiVerticalRhythmSmall"
```

Comment:
A couple things bother me about this button... the white background feels out of place because it doesn't match the background of the panel it's in, and the hover state doesn't feel very noticeable to me:

![image](https://cloud.githubusercontent.com/assets/1238659/25756106/68ead1a0-317a-11e7-84d1-a650cb7645ba.png)

![image](https://cloud.githubusercontent.com/assets/1238659/25756108/6aac6b3e-317a-11e7-8723-ab08d982bc53.png)

I think we should make the background semi-transparent, so it will adjust to the background of the container. And then what if we did a background change to make the hover state more noticeable? This is similar to how this button on elastic.co behaves:

![image](https://cloud.githubusercontent.com/assets/1238659/25756900/5a4a49a2-317d-11e7-846c-3eed55db1c0c.png)

![image](https://cloud.githubusercontent.com/assets/1238659/25756902/5bd3f0f2-317d-11e7-978a-1758833906af.png)

I just threw this together to communicate the idea, I'm sure it could be improved:

![image](https://cloud.githubusercontent.com/assets/1238659/25756702/9ea4cd1c-317c-11e7-93d9-8be7966a68c2.png)

![image](https://cloud.githubusercontent.com/assets/1238659/25757013/bda7a29c-317d-11e7-94e8-4b08338a62a6.png)

Works on white too:

![image](https://cloud.githubusercontent.com/assets/1238659/25757061/e5574450-317d-11e7-868c-130695d85545.png)

![image](https://cloud.githubusercontent.com/assets/1238659/25757063/e719f6ac-317d-11e7-852e-561712e7a79a.png)

Here's the SCSS I used:

```scss
.kuiButton--secondary {
  color: $globalLinkColor !important;
  border: solid 1px $globalColorBlue;
  background-color: rgba($globalColorWhite, 0.4);

  @include kuiButtonHoverAndActive {
    color: darken($globalLinkColor, 10%) !important;
    border: solid 1px darken($globalColorBlue, 10%);
    background-color: rgba(lighten($globalLinkColor, 50%), 0.1);
  }
}
```